src/LanguageServer/Protocol.TestUtilities/AbstractLspMiscellaneousFilesWorkspaceTests.cs

```diff
@@ -45,14 +49,13 @@ void M()
     [Theory, CombinatorialData]
     public async Task TestLooseFile_Changed(bool mutatingLspWorkspace)
     {
-
         // Create a server that supports LSP misc files and verify no misc files present.
         await using var testLspServer = await CreateTestLspServerAsync(string.Empty, mutatingLspWorkspace, new InitializationOptions { ServerKind = WellKnownLspServerKinds.CSharpVisualBasicLspServer });
 
-        var miscWorkspace = testLspServer.GetManager().GetTestAccessor().GetLspMiscellaneousFilesWorkspace();
-        testLspServer.TestWorkspace.GetService<LspWorkspaceRegistrationService>().Register(miscWorkspace);
+        //var miscWorkspace = testLspServer.GetManager().GetTestAccessor().GetLspMiscellaneousFilesWorkspace();
+        //testLspServer.TestWorkspace.GetService<LspWorkspaceRegistrationService>().Register(miscWorkspace);
```

Comment:
Oops gotta remove this, but somehow the tests are still passing?

Reply:
isn't it covered by the workspace automatically being registered by the workspace event listener?  I wouldn't think you need to even have this line